onnxruntime/python/tools/kernel_explorer/kernels/skip_layer_norm.cc

```diff
@@ -38,6 +38,30 @@ class SkipLayerNormSmall : public IKernelExplorer {
   ParamsT params_{};
 };
 
+template <typename T, int ThreadsPerBlock, int VecSize>
+class SkipLayerNormRegular : public IKernelExplorer {
```

Comment:
just double confirm, those are the same as main branch, right?

Reply:
yes, they are the same as main branch